### PR TITLE
Support bind parameters

### PR DESCRIPTION
**Pull request by spadgett**
https://trello.com/c/Sws4p5jU

Requires https://github.com/openshift/origin-web-common/pull/163
See also https://github.com/openshift/origin-web-console/pull/1987

@jeff-phillips-18 FYI

### CODE REVIEW

File: src/components/order-service/order-service.controller.ts [hunk 1]
```diff
@@ -313,6 +351,14 @@ export class OrderServiceController implements angular.IController {
     }
   }
 
+  private updateBindParametersStepVisibility() {
+    // Show the bind parameters step if the bind step if not hidden and the plan has a bind parameter schema.
+    this.bindParametersStep.hidden = this.bindStep.hidden ||
+                                     this.ctrl.bindType === 'none' ||
+                                     !_.has(this.ctrl, 'bindParameterSchema.properties');
+    this.bindParametersStep.allowed = !this.bindParametersStep.hidden && this.bindStep.valid;
```
Comment (jeff-phillips-18) [inline src/components/order-service/order-service.controller.ts]: nit: don't really need to check hidden here, if it is hidden then allowed will never be used.

File: src/components/order-service/order-service.controller.ts [hunk 2]
```diff
@@ -174,7 +196,17 @@ export class OrderServiceController implements angular.IController {
 
     this.validityWatcher = this.$scope.$watch("$ctrl.forms.bindForm.$valid", (isValid: any, lastValue: any) => {
       this.bindStep.valid = isValid;
-      this.reviewStep.allowed = this.bindStep.valid;
+      this.bindParametersStep.allowed = !this.bindParametersStep.hidden && isValid;
```
Comment (jeff-phillips-18) [inline src/components/order-service/order-service.controller.ts]: nit: don't really need to check hidden here, if it is hidden then allowed will never be used.

**Comment by spadgett** [comment]
Needs a common bump :(

**Comment by spadgett** [comment]
Sorry I shouldn't have removed WIP without the common release

**Pull request merged**